Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -21,7 +26,9 @@ def gistify_logs(f)
       files["tap.out"] = { :content => tap }
     end
 
-    url = create_gist(files)
+    # Description formatted to work well as page title when viewing gist
+    descr = "#{f.name} on #{MACOS_FULL_VERSION} - Homebrew build failure "
```

Comment:
:+1:, Agree with Baptiste. Use `gist-logs` quite a bit to compare my successful build against someone's unsuccessful build as it's easier than opening 7 files side-by-side in a text editor heh.